instance/models/openedx_instance.py

```diff
@@ -281,6 +282,19 @@ def first_activated(self):
         except models.ObjectDoesNotExist:
             return None
 
+    @property
+    def most_recently_archived(self):
+        """
+        Returns the date the instance has been most recently archived
+        :return: Union[None, datetime]
+        """
+        relevant_log_regex = r'Archiving instance finished.'
```

Comment:
Is the raw-string indicator needed?

```suggestion
        relevant_log_regex = 'Archiving instance finished.'
```

Reply:
No but since people commonly use raw strings for regexes to deal with escaping I like to do that even if it is not strictly speaking needed. That way if someone later on changes the regex string they will not need to change the type.